src/frontend/src/common/components/library/data_subview/components/createTreeModal/index.tsx

```diff
@@ -40,33 +43,35 @@ interface Props {
   failedSamples: any[];
   open: boolean;
   onClose: () => void;
-  setCreateTreeFailed: (hasFailed: boolean) => void;
+  handleCreateTreeFailed: () => void;
+  handleSetCreateTreeStarted: () => void;
 }
 
 export const CreateTreeModal = ({
   sampleIds,
   failedSamples,
   open,
   onClose,
-  setCreateTreeFailed,
+  handleCreateTreeFailed,
+  handleSetCreateTreeStarted,
 }: Props): JSX.Element => {
   const [treeName, setTreeName] = useState<string>("");
   const [isTreeNameTooLong, setTreeNameTooLong] = useState<boolean>(false);
   const [isTreeBuildDisabled, setTreeBuildDisabled] = useState<boolean>(false);
-  const [treeType, setTreeType] = useState<string>("");
-  const [isContextual, setContextual] = useState<boolean>(true);
-  const [isLocal, setLocal] = useState<boolean>(false);
+  const [treeType, setTreeType] = useState<string>("TARGETED");
```

Comment:
I'm unclear on why we need a `treeType` and then also bools of `isTargeted` and `isNonContextualized` since those are just other ways to tell us the tree type, right? It's somewhat a matter of taste, but I would prefer to see tiny helper funcs get defined if we absolutely need the bools. Things like
```isTreeTargeted(treeType) --> true/false```

However, where I'm seeing those bools below, it's currently like
```checked={isTargeted}```
and I think that could just turn into
```checked={treeType === "TARGETED"} // or better yet, === TREE_TYPES.TARGETED, see my other comment```

I'm being nit-picky here, but I'm always worried when I see multiple sources of truth. It's not happening now, but if `treeType` gets out of sync with its various interpreted bools, we could get weird bugs.

Reply:
i agree, i'll change it to keep the single source of truth

Reply:
removed these vars